hazelcast/src/main/java/com/hazelcast/ClientMain.java

```diff
@@ -12,7 +12,7 @@ public class ClientMain {
 
     public static void main(String[] args) {
         System.setProperty("hazelcast.tpc.enabled", "true");
-        System.setProperty("hazelcast.tpc.eventloop.count", "" + Runtime.getRuntime().availableProcessors());
+        System.setProperty("hazelcast.tpc.eventloop.count", "" + 10);
```

Comment:
minor: why do we change that?

Reply:
https://github.com/hazelcast/hazelcast/pull/23253/commits/695de1aef72f18ec77dc2152e1a1f91dae40ad80

Reply:
To make sure that you run as many TPC threads as there are cores.